test/unit/helper/labelGenerator_default.js

```diff
@@ -175,14 +175,14 @@ module.exports.tests.default_country = function(test, common) {
     t.end();
   });
 
-  test('country', function(t) {
+  test('country layer labels should only use the `country` field and not the `name`', function(t) {
     var doc = {
-      'name': 'country name',
+      'name': 'source country name',
       'layer': 'country',
       'country_a': 'country code',
-      'country': 'country name'
+      'country': 'hierarchy country name'
     };
-    t.equal(generator(doc),'country name');
+    t.equal(generator(doc),'hierarchy country name');
```

Comment:
I think being able to search for a record using the label is important so in the interest of preserving that we should keep it this way for now. I imagine a lot of this will change when we start looking at Alt-Names and internationalization, so we can revisit it then and figure out the "right" approach.